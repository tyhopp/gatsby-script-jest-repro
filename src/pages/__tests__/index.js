import React from "react";
import renderer from "react-test-renderer";
import IndexPage from "../index";

it(`works`, () => {
  expect(true).toBe(true);
});

it(`renders correctly`, () => {
  const tree = renderer.create(<IndexPage />).toJSON();
  expect(tree).toMatchSnapshot();
});
