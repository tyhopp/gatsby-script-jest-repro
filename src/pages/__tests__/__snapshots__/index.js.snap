// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<main
  style={
    Object {
      "color": "#232129",
      "fontFamily": "-apple-system, Roboto, sans-serif, serif",
      "padding": 96,
    }
  }
>
  <title>
    Home Page
  </title>
  <h1
    style={
      Object {
        "marginBottom": 64,
        "marginTop": 0,
        "maxWidth": 320,
      }
    }
  >
    Congratulations
    <br />
    <span
      style={
        Object {
          "color": "#663399",
        }
      }
    >
      â€” you just made a Gatsby site! 
    </span>
    <span
      aria-label="Party popper emojis"
      role="img"
    >
      ðŸŽ‰ðŸŽ‰ðŸŽ‰
    </span>
  </h1>
  <p
    style={
      Object {
        "marginBottom": 48,
      }
    }
  >
    Edit 
    <code
      style={
        Object {
          "backgroundColor": "#FFF4DB",
          "borderRadius": 4,
          "color": "#8A6534",
          "fontSize": "1.25rem",
          "padding": 4,
        }
      }
    >
      src/pages/index.js
    </code>
     to see this page update in real-time.
     
    <span
      aria-label="Sunglasses smiley emoji"
      role="img"
    >
      ðŸ˜Ž
    </span>
  </p>
  <ul
    style={
      Object {
        "marginBottom": 96,
        "paddingLeft": 0,
      }
    }
  >
    <li
      style={
        Object {
          "color": "#8954A8",
          "fontSize": 16,
          "fontWeight": "bold",
          "listStyleType": "none",
          "marginBottom": 24,
          "verticalAlign": "5%",
        }
      }
    >
      <a
        href="https://www.gatsbyjs.com/docs/?utm_source=starter&utm_medium=start-page&utm_campaign=minimal-starter"
        style={
          Object {
            "color": "#8954A8",
            "fontSize": 16,
            "fontWeight": "bold",
            "verticalAlign": "5%",
          }
        }
      >
        Documentation
      </a>
    </li>
    <li
      style={
        Object {
          "color": "#E95800",
          "fontSize": 24,
          "fontWeight": 300,
          "marginBottom": 30,
          "maxWidth": 560,
        }
      }
    >
      <span>
        <a
          href="https://www.gatsbyjs.com/docs/tutorial/?utm_source=starter&utm_medium=start-page&utm_campaign=minimal-starter"
          style={
            Object {
              "color": "#8954A8",
              "fontSize": 16,
              "fontWeight": "bold",
              "verticalAlign": "5%",
            }
          }
        >
          Tutorial
        </a>
        <p
          style={
            Object {
              "color": "#232129",
              "fontSize": 14,
              "lineHeight": 1.25,
              "marginBottom": 0,
              "marginTop": 10,
            }
          }
        >
          A great place to get started if you're new to web development. Designed to guide you through setting up your first Gatsby site.
        </p>
      </span>
    </li>
    <li
      style={
        Object {
          "color": "#1099A8",
          "fontSize": 24,
          "fontWeight": 300,
          "marginBottom": 30,
          "maxWidth": 560,
        }
      }
    >
      <span>
        <a
          href="https://www.gatsbyjs.com/docs/how-to/?utm_source=starter&utm_medium=start-page&utm_campaign=minimal-starter"
          style={
            Object {
              "color": "#8954A8",
              "fontSize": 16,
              "fontWeight": "bold",
              "verticalAlign": "5%",
            }
          }
        >
          How to Guides
        </a>
        <p
          style={
            Object {
              "color": "#232129",
              "fontSize": 14,
              "lineHeight": 1.25,
              "marginBottom": 0,
              "marginTop": 10,
            }
          }
        >
          Practical step-by-step guides to help you achieve a specific goal. Most useful when you're trying to get something done.
        </p>
      </span>
    </li>
    <li
      style={
        Object {
          "color": "#BC027F",
          "fontSize": 24,
          "fontWeight": 300,
          "marginBottom": 30,
          "maxWidth": 560,
        }
      }
    >
      <span>
        <a
          href="https://www.gatsbyjs.com/docs/reference/?utm_source=starter&utm_medium=start-page&utm_campaign=minimal-starter"
          style={
            Object {
              "color": "#8954A8",
              "fontSize": 16,
              "fontWeight": "bold",
              "verticalAlign": "5%",
            }
          }
        >
          Reference Guides
        </a>
        <p
          style={
            Object {
              "color": "#232129",
              "fontSize": 14,
              "lineHeight": 1.25,
              "marginBottom": 0,
              "marginTop": 10,
            }
          }
        >
          Nitty-gritty technical descriptions of how Gatsby works. Most useful when you need detailed information about Gatsby's APIs.
        </p>
      </span>
    </li>
    <li
      style={
        Object {
          "color": "#0D96F2",
          "fontSize": 24,
          "fontWeight": 300,
          "marginBottom": 30,
          "maxWidth": 560,
        }
      }
    >
      <span>
        <a
          href="https://www.gatsbyjs.com/docs/conceptual/?utm_source=starter&utm_medium=start-page&utm_campaign=minimal-starter"
          style={
            Object {
              "color": "#8954A8",
              "fontSize": 16,
              "fontWeight": "bold",
              "verticalAlign": "5%",
            }
          }
        >
          Conceptual Guides
        </a>
        <p
          style={
            Object {
              "color": "#232129",
              "fontSize": 14,
              "lineHeight": 1.25,
              "marginBottom": 0,
              "marginTop": 10,
            }
          }
        >
          Big-picture explanations of higher-level Gatsby concepts. Most useful for building understanding of a particular topic.
        </p>
      </span>
    </li>
    <li
      style={
        Object {
          "color": "#8EB814",
          "fontSize": 24,
          "fontWeight": 300,
          "marginBottom": 30,
          "maxWidth": 560,
        }
      }
    >
      <span>
        <a
          href="https://www.gatsbyjs.com/plugins?utm_source=starter&utm_medium=start-page&utm_campaign=minimal-starter"
          style={
            Object {
              "color": "#8954A8",
              "fontSize": 16,
              "fontWeight": "bold",
              "verticalAlign": "5%",
            }
          }
        >
          Plugin Library
        </a>
        <p
          style={
            Object {
              "color": "#232129",
              "fontSize": 14,
              "lineHeight": 1.25,
              "marginBottom": 0,
              "marginTop": 10,
            }
          }
        >
          Add functionality and customize your Gatsby site or app with thousands of plugins built by our amazing developer community.
        </p>
      </span>
    </li>
    <li
      style={
        Object {
          "color": "#663399",
          "fontSize": 24,
          "fontWeight": 300,
          "marginBottom": 30,
          "maxWidth": 560,
        }
      }
    >
      <span>
        <a
          href="https://www.gatsbyjs.com/cloud?utm_source=starter&utm_medium=start-page&utm_campaign=minimal-starter"
          style={
            Object {
              "color": "#8954A8",
              "fontSize": 16,
              "fontWeight": "bold",
              "verticalAlign": "5%",
            }
          }
        >
          Build and Host
        </a>
        <span
          aria-label="New Badge"
          style={
            Object {
              "backgroundColor": "#088413",
              "border": "1px solid #088413",
              "borderRadius": 4,
              "color": "#fff",
              "display": "inline-block",
              "fontSize": 11,
              "fontWeight": "bold",
              "letterSpacing": 1,
              "lineHeight": 1,
              "marginLeft": 10,
              "padding": "4px 6px",
              "position": "relative",
              "top": -2,
            }
          }
        >
          NEW!
        </span>
        <p
          style={
            Object {
              "color": "#232129",
              "fontSize": 14,
              "lineHeight": 1.25,
              "marginBottom": 0,
              "marginTop": 10,
            }
          }
        >
          Now youâ€™re ready to show the world! Give your Gatsby site superpowers: Build and host on Gatsby Cloud. Get started for free!
        </p>
      </span>
    </li>
  </ul>
  <img
    alt="Gatsby G Logo"
    src="data:image/svg+xml,%3Csvg width='24' height='24' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M12 2a10 10 0 110 20 10 10 0 010-20zm0 2c-3.73 0-6.86 2.55-7.75 6L14 19.75c3.45-.89 6-4.02 6-7.75h-5.25v1.5h3.45a6.37 6.37 0 01-3.89 4.44L6.06 9.69C7 7.31 9.3 5.63 12 5.63c2.13 0 4 1.04 5.18 2.65l1.23-1.06A7.959 7.959 0 0012 4zm-8 8a8 8 0 008 8c.04 0 .09 0-8-8z' fill='%23639'/%3E%3C/svg%3E"
  />
</main>
`;
